src/App.vue

```diff
@@ -80,17 +83,44 @@ export default {
   },
 
   computed: {
-    authAPIURL () {
+    /** The Auth API string. */
+    authApiUrl (): string {
       return sessionStorage.getItem('AUTH_API_URL')
     },
 
-    showLoadingContainer () {
-      return !this.dataLoaded && !this.dashboardUnavailableDialog && !this.accountAuthorizationDialog
+    /** The Business ID string. */
+    businessId (): string {
+      return sessionStorage.getItem('BUSINESS_ID')
+    },
+
+    /** Is True if loading container should be shown, else False. */
+    showLoadingContainer (): boolean {
+      return (!this.dataLoaded && !this.dashboardUnavailableDialog && !this.accountAuthorizationDialog)
+    },
+
+    /** Is True if route is Signin, else False. */
+    isSigninRoute (): boolean {
+      return Boolean(this.$route.name === 'signin')
+    },
+
+    /** Is True if user is not authenticated, else False. */
+    isNotAuthenticated (): boolean {
+      // FUTURE: also check that token isn't expired!
+      return Boolean(!sessionStorage.getItem('KEYCLOAK_TOKEN'))
```

Comment:
Not 100% sure on the context of the method, but would it make more sense to be `isAuthenticated` and return true if they are?

Reply:
I originally did this to simplify where this is used (ie, check for a condition instead of !condition) but I have since changed my mind and changed this to "isAuthenticated". Great minds think alike :)